runelite-client/src/main/resources/net/runelite/client/runelite.properties

```diff
@@ -5,4 +5,7 @@ runelite.discord.appid=409416265891971072
 runelite.discord.invite=https://discord.gg/R4BQ8tU
 runelite.github.link=https://github.com/runelite
 runelite.wiki.link=https://github.com/runelite/runelite/wiki
-runelite.patreon.link=https://www.patreon.com/runelite
\ No newline at end of file
+runelite.patreon.link=https://www.patreon.com/runelite
+runelite.wiki.troubleshooting.link=https://github.com/runelite/runelite/wiki/Troubleshooting-problems-with-the-client
+runelite.wiki.building.link=https://github.com/runelite/runelite/wiki/Building-with-IntelliJ-IDEA#client-failing-to-start
```

Comment:
Does this work? github is rendering it as a comment

Reply:
this works correctly